mappings/net/minecraft/class_5459.mapping

```diff
@@ -1,4 +1,24 @@
 CLASS net/minecraft/class_5459
+	METHOD method_30576 getLargestRectangle ([I)Lcom/mojang/datafixers/util/Pair;
+		COMMENT Gets the largest rectangle within a histogram, where the vertical bars each have
```

Comment:
```suggestion
	METHOD method_30576 findLargestRectangle ([I)Lcom/mojang/datafixers/util/Pair;
		COMMENT Finds the largest rectangle within a histogram, where the vertical bars each have
```

`find` could be better since the largest rectangle is not obvious from a histogram.

Reply:
Lol, leetcode has bad names as well :laughing: Let me check if there is other associated renames